pkg/controller/agent/pod.go

```diff
@@ -381,15 +387,22 @@ func getAssociatedFleetServer(params Params) (commonv1.Associated, error) {
 	return &fs, nil
 }
 
-func trustCAScript(caPath string) string {
+func trustCAScript(version version.Version, caPath string) string {
+	sharedCAPath := "/etc/pki/ca-trust/source/anchors/"
+	updateCmd := "update-ca-trust"
+	if version.Major == 8 {
```

Comment:
Do we want `>=` 8?  Otherwise, we'd have to update this again when work began on v9.

Reply:
Well, I guess it's a question of whether we think that the next version is going to go back to CentOS style path/cmd, as that's how it will currently be handled.

Reply:
No, this is a good observation. Even if there is not other major after 8 we should not box ourselves in here.